README.md

```diff
@@ -1,23 +1,42 @@
 # UE4SSCPPTemplate
 
-To setup your project, run the following command in this directory, replacing "YourModName" with the name of your mod (no spaces or special characters, please!):
+This repository includes some custom `xmake` tasks to assist in your mod development.
 
-```bash
-.\new_mod_setup.bat YourModName
-```
+- `xmake newmod`
+- `xmake installmod`
+- `xmake bi`
+- `xmake ue4ss`
 
-This will create a directory vsxmake2022 with a `.sln` file that you can open.
+## `xmake newmod [-r, --regen] name`
 
-Then, when you are ready to build and install your mod into your game, run the following command in this directory:
+The `newmod` command will ensure you have the UE4SS repository cloned and then bootstrap new mod creation.
 
-```bash
-.\build_and_install_mod.bat YourModName "Path\To\Your\Game\UE4SS\Install\Directory" Build__Configuration 
-```
+Running `xmake newmod CoolMod` will generate `/Mods/CoolMod/...` in your repository and automatically link it with the xmake system. If you have VS2022 installed then the `newmod` command will also automatically generate a VS project which can be located at `/vsxmake2022/UE4SSCppTemplate.sln`.
 
-For example:
+## `xmake installmod [-d, --exedir] name`
 
-```bash
-.\build_and_install_mod.bat MyAwesomeMod "C:\Program Files (x86)\Steam\steamapps\common\Deep Rock Galactic\FSD\Binaries\Win64" Game__Shipping__Win64
-```
+The `installmod` command will copy your build output to a specific game directory. The configuration of the build you want to install is derived from the current xmake configuration. If you want to specify a specific configuration to install, then you can change the xmake configuration by running `xmake f -m "Game__Debug__Win64"`.
```

Comment:
> The configuration of the build you want to install is derived from the current xmake configuration.
This means the RE-UE4SS xmake config right?

Reply:
Added clarification about the global nature of xmake config -m "Mode"